scopus-import/src/main/java/no/sikt/nva/scopus/conversion/AffiliationGenerator.java

```diff
@@ -82,39 +75,47 @@ private List<CorporationWithContributors> fillCristinOrganizationData(
     }
 
     private CorporationWithContributors addCristinOrganisationData(CorporationWithContributors corporation) {
-        return nonNull(corporation.getCristinOrganizationId())
-                   ? fetCristinOrganizationsById(corporation)
-                   : searchForCristinOrganizationByName(corporation);
+        var cristinOrganization = fetchCristinOrganizationsById(corporation)
+            .or(() -> searchForCristinOrganizationByName(corporation))
+            .or(() -> searchForCristinOrganizationByCountry(corporation));
+
+        return cristinOrganization.isPresent()
+                   ? corporation.copy().withCristinCorporations(List.of(cristinOrganization.get())).build()
+                   : corporation;
+    }
+
+    private Optional<CristinOrganization> searchForCristinOrganizationByCountry(
```

Comment:
What does this do? get the affiliation country and search for a org with that name? is that correct?

Reply:
Almost, countries in Cristin are like organizations. They are returned when you are searching for organization with for example name "Sweden" or "Norway". We want all contributors created under import to have verified affiliation. If we could not fetch organization for contributor, we are trying to fetch contributor country. So when importing this publication, we end up with contributor that has country as affiliation:

![image](https://github.com/user-attachments/assets/bb9870e7-c0ce-4746-963f-3fa9c352f8f0)

Reply:
We are not really interested in non Norwegian contributor organizations, we need only to know if there are some international contributors on publication (important for getting more nvi points)